src/website/src/app/documentation/demos/progress-bars/progress-bar-component.demo.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  ~ Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<div class="progbar-examples">
+    <p>
+        Clarity also has a component <code class="clr-code">clrProgressBar</code>. That will add accessibility support for screen readers.
+    </p>
+
+    <ul class="list">
+        <li><strong>value</strong> - This attribute specifies how much of the task that has been completed. 
+        It must be a valid floating point number between 0 and max</li>
+        <li><strong>max</strong> - This attribute describes how much work the task indicated by the progress
+        element requires. The max attribute, if present, must have a value greater than zero and be a valid 
+        floating point number. The default value is 100.</li>
+        <li><strong>displayval</strong> - Overwrite the default value representation. Default is [<strong>value</strong>]% </li>
+        <li><strong>liveRegion</strong> - Change the default <code class="clr-code">aria-live</code> value. Valid values are: <strong>polite</strong>, <strong>assertive</strong>, <strong>off</strong></li>
+    </ul>
+
+    <h4>Examples</h4>
+    <clr-code-snippet [clrCode]="example"></clr-code-snippet>
+    <clr-progress-bar value="40"></clr-progress-bar>
+   
+    <br/>
+
+    <clr-code-snippet [clrCode]="example1"></clr-code-snippet>
+    <clr-progress-bar value="65" class="labeled"></clr-progress-bar>
+
+    <br/>
+    
+    <clr-code-snippet [clrCode]="example2"></clr-code-snippet>
+    <div class="progress-block">
+        <label>Text on the left</label>
```

Comment:
You can use this example to associate a `label` to the progress component via `id` and `for`.